src/proto.rs

```diff
@@ -0,0 +1,5 @@
+#[cfg(any(feature = "sources-vector", feature = "sinks-vector"))]
+use crate::event::proto as event;
```

Comment:
This is needed because the protobuf definition line:

```
import "event.proto";
```

gets turned into Rust code:

```
use super::event
```